register-success.html

```diff
@@ -0,0 +1,45 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Register Success</title>
+  <!-- Favicon -->
+  <link rel="icon" type="image/x-icon" href="assets/icons/favicon.ico">
+  <!-- Google Font -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Aoboshi+One&family=Open+Sans:wght@500;600;700;800&family=Poppins:wght@300;400;500;600;700;800&display=swap" rel="stylesheet">
+  <!-- Style -->
+  <link rel="stylesheet" href="assets/css/style.css">
+  <!-- Tailwind -->
+  <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+  <div class="
+    bg-[url('/assets/images/blue-background.webp')] 
+    bg-no-repeat bg-center bg-cover 
+    w-full h-auto lg:h-screen 
+    flex justify-center items-center 
+    px-4 py-20"
+  >
+  <div class="flex flex-col bg-white justify-center items-center w-full max-w-[1064px] h-auto rounded-[24px] md:rounded-[40px] py-[130px] md:py[102px] lg:py-[114px] px-3 gap-y-4 select-none">
+    <div class="w-full lg:w-auto flex-auto lg:flex-1 flex justify-center items-center self-center" >
+      <img src="assets/images/register-success.webp" class="w-[105px]" alt="Register Success">
+    </div>
+
+    <div class="flex-1 flex flex-col justify-center items-center mb-5">
+      <h1 class="text-[#272C49] text-[27px] md:text-[33px] text-center font-extrabold font-opensans mb-4">You have been Registered!</h1>
+      <p class="text-[#272C49] text-sm md:text-base text-center mb-1">Jangan Lupa untuk cek inbox email mu untuk<br>mengaktivasi akun</p>
```

Comment:
Untuk yang tag br hanya muncul saat didesktop ya Istika, sugesti ku gini yaa:
```suggestion
      <p class="text-[#272C49] text-sm md:text-base text-center mb-1">Jangan Lupa untuk cek inbox email mu untuk <br class="hidden md:block">mengaktivasi akun</p>
```